rpmbuild/bin/copr-builder

```diff
@@ -0,0 +1,222 @@
+#! /usr/bin/python3
+
+"""
+Allow users to control their Copr builder instance
+https://github.com/fedora-copr/debate/tree/main/user-ssh-builders
+"""
+
+import argparse
+from datetime import datetime, timedelta
+from contextlib import suppress
+from copr_common.helpers import timedelta_to_dhms
+
+
+RPMBUILD_PID_PATH = "/var/lib/copr-rpmbuild/pid"
```

Comment:
We seem to have this in config already (helpers.py)

Reply:
Good catch, fixed.